translator/src/main/java/com/adaptivescale/rosetta/translator/DefaultTranslator.java

```diff
@@ -52,11 +49,44 @@ private Column translateColumn(Column column) {
             throw new RuntimeException("There is no match for column name: " + column.getName() + " and type: " + column.getTypeName() + ".");
         }
 
+        List<TranslationAttributeModel> translationAttributes = translationMatrix.findByTranslationAttributesByTranslationIds(translationModel.getId());
```

Comment:
Can we call the singleton with getInstance method instead of storing it to prevent multi-thread issues?